src/user-management/user-management.ts

```diff
@@ -400,4 +405,19 @@ export class UserManagement {
       options,
     );
   }
+
+  async sendInvitation(payload: SendInvitationOptions): Promise<Invitation> {
+    const { data } = await this.workos.post<
+      InvitationResponse,
+      any,
```

Comment:
ooc what's going on with this `any` here? i don't think i understand what this is doing

Reply:
I could look into it but it's just copy / paste from other examples